sycl/source/detail/graph_impl.cpp

```diff
@@ -730,10 +730,13 @@ void executable_command_graph::finalizeImpl() {
         info::graph_support_level::native;
 
 #if FORCE_EMULATION_MODE
-    // Above query should still succeed in emulation mode, but ignore the
-    // result and use emulation.
-    CmdBufSupport = false;
+    setEmulationModeForced();
```

Comment:
`setEmulationModeForced(bool Forced)` should be passed a bool parameter, `setEmulationModeForced(true)`